qiskit/primitives/backend_estimator_v2.py

```diff
@@ -87,7 +87,18 @@ class BackendEstimatorV2(BaseEstimatorV2):
     precludes doing any provider- or backend-specific optimizations.
 
     This class does not perform any measurement or gate mitigation, and, presently, is only
-    compatible with Pauli-based observables.
+    compatible with Pauli-based observables. More formally, given an observable of the type
+    :math:`O=\sum_{i=1}^Na_iP_i`, where :math:`a_i` is a complex number and :math:`P_i` is a
+    Pauli operator, the estimator calculates the expectation :math:`E(P_i)` of each :math:`P_i`
+    and finally calculates the expectation value of :math:`O` as
+    :math:`E(O)==\sum_{i=1}^Na_iE(P_i)`. The reported ``std`` is calculated as
+
+    .. math::
+
+        \frac{\sum_{i=1}^{n}|a_i|\sqrt{\textrm{Var}\big(P_i\big)}}{\sqrt{N}}\:,
```

Comment:
what is `\:` ? I've never used that
```suggestion
        \frac{\sum_{i=1}^{n}|a_i|\sqrt{\textrm{Var}\big(P_i\big)}}{\sqrt{N}},
```

Reply:
what's the link to that paper which suggests this? it's not intuitive to me.

Reply:
Eq. 5 of https://arxiv.org/abs/1908.06942. I added a reference to it in the docstring

Reply:
`\:` leaves a blank space. I normally use it when I have a comma or a break at the end of a full-line equation